expression/builtin_info.go

```diff
@@ -803,6 +803,9 @@ func (b *builtinTiDBDecodePlanSig) evalString(row chunk.Row) (string, bool, erro
 		return "", isNull, err
 	}
 	planTree, err := plancodec.DecodePlan(planString)
+	if err != nil {
+		return planString, false, nil
+	}
 	return planTree, false, err
 }
```

Comment:
```suggestion
	planTree, _ := plancodec.DecodePlan(planString)
	return planTree, false, nil
}
```

Reply:
Ops, look like `make check` will get a warning for doesn't check the error, forgot that.